data/org.freedesktop.portal.Settings.xml

```diff
@@ -43,6 +43,24 @@
           Unknown values should be treated as 0 (no preference).
         </para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>org.freedesktop.appearance window-decorations s</term>
+        <listitem><para>
+          Arrangement of buttons on the titlebar.
+          Supported values are:
+          <simplelist>
+            <member>menu: application content, or titlebar</member>
+            <member>minimize: the minimize button</member>
+            <member>maximize: the maximize button</member>
+            <member>close: the close button</member>
+          </simplelist>
+          Unknown values should be ignored. If a value is duplicated, it should be drawn again in the indicated area.
+          Read from left-to-right; a colon separates the left corner from the right corner, and button names are comma-separated.
+          If menu is not supplied, assume all buttons are to be drawn on the right.
+        </para></listitem>
+      </varlistentry>
+    </variablelist>
```

Comment:
```suggestion
```

The closing tag is already present after

Reply:
Whoops. Thanks.